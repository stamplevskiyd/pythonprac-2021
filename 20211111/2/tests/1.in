(1, 1), (3, 3), (-1, -1)
"qwerty
tuple((i,j) for i in range(2,4) for j in range(7,9) if i + j < 11)

