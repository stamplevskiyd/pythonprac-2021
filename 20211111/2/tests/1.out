Not a triangle
Invalid input
0.50
